src/plant/plant.repository.ts

```diff
@@ -22,7 +22,7 @@ export class PlantRepository extends Repository<Plant> {
       .leftJoin('monitorLocation.stackPipe', 'locationStack')
       .where('plant.oris_code = :orisCode', { orisCode })
       .andWhere('locationUnit.unit_id IN(:...unitIds)', { unitIds })
```

Comment:
unit_id should be changed to unitId since that's sort of like the "unit_name" column for that one, similar to stackpipe.